pkg/apis/scheduling/helpers_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"testing"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
This import is usually not named, just leave it as "k8s.io/api/core/v1" and use v1 in the code below as the package name.

Reply:
Thanks @ahg-g I will update the code and remove the name for corev1 "k8s.io/api/core/v1" . 
How do I know if PR https://github.com/kubernetes/kubernetes/pull/76990/files#r286539095  gets merged first do I need to keep track of it?

Reply:
yeah, I think you will just need to keep an eye on it. Most people are at kubecon hence the delay in reviews, hopefully some progress will be made this week.